src/components/pages/TrophyRoom/Leaderboard.js

```diff
@@ -10,7 +10,7 @@ const Leaderboard = () => {
   useEffect(() => {
     //Getting data from backend for leaderboard
     getLeaderboard(authState).then(res => {
-      console.log(res);
+      console.log('res from getLeaderBoard', res);
```

Comment:
we should remove console logs before pushing to [main] if possible.

not to be a stickler, but Ryan Hamblin destroyed my code from last labs in the guided project this week bc i'd left in so many console.logs lol. it was mortifying!

Reply:
Good catch! I totally forgot to look for those.